website/docs/r/ingress.html.markdown

```diff
@@ -21,15 +21,15 @@ resource "kubernetes_ingress" "example_ingress" {
 
   spec {
     backend {
-      service_name = "MyApp1"
+      service_name = "myapp-1"
       service_port = 8080
     }
 
     rule {
       http {
         path {
           backend {
-            service_name = "MyApp1"
+            service_name = "myapp-1"
```

Comment:
Theres a couple of other spots where this camel case label is used – I agree with changing the convention to be lowercase but I would update the rest of this file to be consistent. You might want to do a search for `MyApp1` and `MyApp2` to find them all.

Reply:
Good catch, @jrhouston. Updated pod labels in the examples. Thank you!